apm-agent-plugin-sdk/pom.xml

```diff
@@ -62,6 +62,14 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Yes, the OTel API requires Java 8. It uses default interface methods, for example.

Reply:
Since `co.elastic.apm.agent.sdk.ElasticApmInstrumentation` is in the module, Java 7 is (effectively) no longer supported after this PR. Maybe source and target should be bumped in the parent POM